game/unicode.cc

```diff
@@ -98,7 +96,7 @@ std::string UnicodeUtil::convertToUTF8 (std::string_view str, std::string _filen
 	return ret;
 }
 
-bool UnicodeUtil::removeUTF8BOM(std::string_view str) {
+bool UnicodeUtil::removeUTF8BOM(std::string_view& str) {
```

Comment:
Reference needed?

Reply:
Yes because it potentially modifies the string_view to a substring of itself.